doc/cookbook/source/examples/evaluation/cross_validation.rst

```diff
@@ -0,0 +1,90 @@
+================
+Cross Validation
+================
+
+Cross validation allows to estimate an algorithm's performance on unseen data.
+
+It is based on training the model on a subset of the data, and testing on the
+remaining data.
+
```

Comment:
minor: no paragraph break, combine with previous

Reply:
Sure, Fixed on the next commit